authorization/1.6/_index.md

```diff
@@ -0,0 +1,38 @@
+---
+title: "Jakarta Authorization 1.6"
```

Comment:
The version of JACC in Java EE 8 is 1.5, I don't know why this is 1.6 but I believe it should all be 1.5.

Reply:
@bshannon the last version released by Oracle in Maven central was unfortunately set at 1.6:

https://mvnrepository.com/artifact/javax.security.jacc/javax.security.jacc-api/1.6

I discussed this last time with Dmitry for the GlassFish 5.1 release and there we decided to release it as 1.6:

https://mvnrepository.com/artifact/jakarta.security.jacc/jakarta.security.jacc-api